site/docs/configuration.md

```diff
@@ -61,6 +61,10 @@ Iceberg tables support table properties to configure table behavior, like the de
 | commit.retry.min-wait-ms           | 100              | Minimum time in milliseconds to wait before retrying a commit |
 | commit.retry.max-wait-ms           | 60000 (1 min)    | Maximum time in milliseconds to wait before retrying a commit |
 | commit.retry.total-timeout-ms      | 1800000 (30 min) | Maximum time in milliseconds to wait before retrying a commit |
+| commit.status-check.num-retries    | 3                | Number of times to check the commit status before failing     |
```

Comment:
I think this is a bit too unclear because it isn't clear what "check the commit status" or "failing" mean exactly. I think this should be "Number of times to check whether a commit succeeded after a connection is lost before failing due to an unknown commit state".

Reply:
I added a fix for this.